signer/src/stacks/contracts.rs

```diff
@@ -73,8 +73,8 @@ pub struct ReqContext {
     /// [`AsContractCall::validate`] function, but is here for logging and
     /// tracking purposes.
     pub origin: PublicKey,
-    /// The combined public key of all the signers in the current
-    /// signing set.
+    /// This is the bitcoin aggregate key that was output from DKG. It is
```

Comment:
I think we should still specify that this is a public key in the description despite the fact it's clear from the type of the field as well.

```suggestion
    /// This is the bitcoin aggregate public key that was output from DKG. It is
```

Reply:
https://github.com/stacks-network/sbtc/pull/694/commits/c3ceea7f8a43bcdd8ef975170e6c5d9aaf965461